dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04310CreateWorkflowRoles.java

```diff
@@ -0,0 +1,178 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.business.WrapInTransaction;
+import com.dotmarketing.business.RoleAPI;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import com.dotmarketing.util.Logger;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Add the Workflow Roles
+ */
+public class Task04310CreateWorkflowRoles extends AbstractJDBCStartupTask {
+
+    private final String getSystemRole =
+            "select id, role_name, description, role_key, db_fqn, parent, edit_permissions, edit_users, edit_layouts, "
+                    +
+                    "locked, system from cms_role where role_key = 'System' and id = parent";
+
+    private final String selectSystemRoles =
+            "select id, role_name, description, role_key, db_fqn, parent, edit_permissions, edit_users, edit_layouts, "
+                    +
+                    "locked, system from cms_role where parent = ? and parent <> id";
+
+    private final String insertRole =
+            "insert into cms_role (id, role_name, description, role_key, db_fqn, parent, edit_permissions, "
+                    +
+                    "edit_users, edit_layouts, locked, system) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
+
+    private final String AnyoneWhoCanReadRole = "Anyone who can View Content";
+    private final String AnyoneWhoCanReadRoleId = "0f995057-5c35-4ae7-998a-774dda146c63";
+    private final String AnyoneWhoCanEditRole = "Anyone who can Edit Content";
+    private final String AnyoneWhoCanEditRoleId = "617f7300-5c7b-463f-9554-380b918520bc";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)